mosec/server.py

```diff
@@ -69,13 +72,35 @@ def _validate_arguments(
         num,
         max_batch_size,
         start_method,
+        env,
     ):
         def validate_int_ge_1(number, name):
             assert isinstance(
                 number, int
             ), f"{name} must be integer but you give {type(number)}"
             assert number >= 1, f"{name} must be greater than 1"
 
+        def validate_env():
+            if env is None:
+                return
+
+            def validate_str_dict(dictionary: Dict):
+                for k, v in dictionary.items():
+                    if not (isinstance(k, str) and isinstance(v, str)):
+                        return False
+                return True
+
+            assert len(env) == num, "len(env) must equal to num"
+            valid = True
+            if not isinstance(env, List):
+                valid = False
+            elif not all(
+                [lambda x: isinstance(x, Dict) and validate_str_dict(x) for x in env]
```

Comment:
Check your Pylance warning here.

Reply:
can give a screenshot on that? I don't see warning here

Reply:
You created a lambda function but you never call it.